gradle.properties

```diff
@@ -1,21 +1,19 @@
-# Project-wide Gradle settings.
-# IDE (e.g. Android Studio) users:
-# Gradle settings configured through the IDE *will override*
-# any settings specified in this file.
-# For more details on how to configure your build environment visit
-# http://www.gradle.org/docs/current/userguide/build_environment.html
-# Specifies the JVM arguments used for the daemon process.
-# The setting is particularly useful for tweaking memory settings.
-org.gradle.jvmargs=-Xmx2048m -Dfile.encoding=UTF-8
-# When configured, Gradle will run in incubating parallel mode.
-# This option should only be used with decoupled projects. For more details, visit
-# https://developer.android.com/r/tools/gradle-multi-project-decoupled-projects
-# org.gradle.parallel=true
-# AndroidX package structure to make it clearer which packages are bundled with the
-# Android operating system, and which are packaged with your app's APK
-# https://developer.android.com/topic/libraries/support-library/androidx-rn
+
 android.useAndroidX=true
-# Enables namespacing of each library's R class so that its R class includes only the
-# resources declared in the library itself and none from the library's dependencies,
-# thereby reducing the size of the R class for that library
-android.nonTransitiveRClass=true
+android.enableJetifier=true
+GROUP=com.rudderstack.android.integration
+kotlin.code.style=official
+POM_LICENCE_URL=http://opensource.org/licenses/MIT
+org.gradle.jvmargs=-Xmx1536m
+POM_DEVELOPER_ID=Rudderstack
+POM_LICENCE_NAME=Elastic License 2.0 (ELv2)
+VERSION_CODE=5
+POM_DEVELOPER_NAME=Rudderstack, Inc.
+POM_LICENCE_DIST=repo
+VERSION_NAME=1.0.1
```

Comment:
```suggestion
VERSION_NAME=1.0.0
```

Reply:
JFI: Depending on this version, the Sprig-Android integration will be released. So we need to ensure, it points to the correct version.